mozetl/schemas.py

```diff
@@ -0,0 +1,10 @@
+import ujson as json
+import os
+
+from pyspark.sql.types import StructType
+
+directory = os.path.join(os.path.dirname(__file__), 'json')
```

Comment:
Thanks! Turns out pkg_resources.resource_stream() is a ContextManager which returns a file object. Would you like me to patch topline to use a with ... : block or should I leave it alone?